siphon/tests/test_igra2.py

```diff
@@ -0,0 +1,91 @@
+# Copyright (c) 2018 University Corporation for Atmospheric Research/Unidata.
+# Distributed under the terms of the MIT License.
+# SPDX-License-Identifier: MIT
+"""Test IGRA2 upper air dataset access."""
+
+from datetime import datetime
+
+from numpy.testing import assert_almost_equal
+
+from siphon.simplewebservice.igra2 import IGRAUpperAir
+from siphon.testing import get_recorder
+
+
+recorder = get_recorder(__file__)
+
+
+@recorder.use_cassette('igra2_sounding')
```

Comment:
Need to add `igra2_sounding`, in `siphon/tests/fixtures` to the PR so that we have it saved and the tests don't need the network.

Reply:
How do you add the recorder files?

Reply:
`git add siphon/test/fixures/igra2_sounding`. That might depend on how you're interacting with git though. The shorter answer is "However you tell your git client to add an untracked file."

Reply:
I was actually asking how to create the recorder files - I think the problem might be that accessing FTP isn't handled the same way as accessing HTTP files. So when I run the
````
@recorder.use_cassette('igra2_sounding')
def test_igra2()...
````
nothing is created.

Reply:
OH! Those files should be generated automatically, so that does likely mean that FTP is not playing nicely with vcrpy. That's annoying. We can leave it as is for now then and figure something out later.

Reply:
Sounds good. I wonder if there should be a section for FTP access. (not-quite-as-simple webservice?) I know that the IGRA2 maintainers are looking to move to a different mode of delivery, but that is probably years away.

Reply:
Legacy web service? 😁 Eh, we can keep it here, at least until we have more.